docs/mkdocs.yml

```diff
@@ -72,9 +72,12 @@ plugins:
   - search:
       separator: '[\s\-,:!=\[\]()"`/]+|\.(?!\d)|&[lg]t;|(?!\b)(?=[A-Z][a-z])'
   # - meta                         # (insiders) use .meta.yml files
-  - glightbox                    # image zoom
+  - glightbox:                   # image zoom
```

Comment:
How it works? I thought about special class for disabled images

Reply:
You have suggested disabling with classes but still it will be enabled globally. 

I took the opposite approach.
```yaml
plugins:
  - glightbox:
      manual: true # enable manual mode, all images will not be added to the lightbox effect
```
The above code disables lightbox effect for all the image by default. If lightbox effect is needed then it can be enabled by adding a special class. https://blueswen.github.io/mkdocs-glightbox/flexibility/enable-by-image-or-page/

I don't think we need lightbox effect anywhere in our faststream docs. So, I took the approach of disabling it globally.

Reply:
Btw, I added this plugin to use lightbox everywhere 😄 If you disabled it globally, we should go through all documentation and set your special class to all old images

Reply:
It just a 3 pages as I can see

https://faststream.airt.ai/latest/getting-started/asyncapi/hosting/#customizing-asyncapi-documentation
https://faststream.airt.ai/latest/getting-started/opentelemetry/
https://faststream.airt.ai/latest/getting-started/asyncapi/custom/#setup-custom-handler-information

Reply:
I have searched for all the embedded images and added lightbox effect to them. It was in 4 pages I guess.